introduction/Organization.tex

```diff
@@ -35,18 +38,21 @@ \subsection{Organizing Committee --- oc@robocupathome.org}
 
 \begin{itemize}
 	\item \textbf{[Chair]} Maxime St-Pierre (Clearpath Robotics, Canada)
-	\item \textbf{[Co-Chair]} Fabrice Jumel (Université de Lyon; France)
-	\item Angelica Lim (Canada)
-	\item Francisco Javier Rodriguez Lera (Universidad de León, Spain)
-	\item Jeffrey Too Chuan Tan (Nankai University, China)
-	\item Jesus Savage (University of Mexico, Mexico)
-	\item João Cartucho (University of Lisbon, Portugal)
-	\item Luz Martínez (Universidad de Chile, Chile)
-	\item Nathalie Connolly (Concordia University, Canada)
-	\item Raphael Memmesheimer (University of Koblenz, Germany)
+	\item Colm Flanagan (University New South Wales, Australia)
+	\item David Garcia (Colombia)
+	\item Fagner Pimentel (FEI University Center, Brazil)
+	\item Florian Lier (University of Bielefeld, Germany)
```

Comment:
Florian Lier (Bielefeld University, Germany)

Reply:
```suggestion
	\item Florian Lier (Bielefeld University, Germany)
```

Reply:
Really? It has been like that for years!

Reply:
Haha yeh I thought the same. But @johaq went there so I assume he knows?

Reply:
https://en.m.wikipedia.org/wiki/Bielefeld_University

Reply:
No you don’t always add the “of” it’s either part of the university’s name or it isn’t.

Looks like we’ve always had it wrong.

Reply:
No you don’t always add the “of” it’s either part of the university’s name or it isn’t.

Looks like we’ve always had it wrong.